src/test/run-make/raw-dylib/Makefile

```diff
@@ -0,0 +1,18 @@
+# only-windows
+
+-include ../../run-make-fulldeps/tools.mk
+
+all:
+	$(call COMPILE_OBJ,"$(TMPDIR)"/extern_1.obj,extern_1.c)
+	$(call COMPILE_OBJ,"$(TMPDIR)"/extern_2.obj,extern_2.c)
+	$(CC) "$(TMPDIR)"/extern_1.obj -link -dll -out:"$(TMPDIR)"/extern_1.dll
+	$(CC) "$(TMPDIR)"/extern_2.obj -link -dll -out:"$(TMPDIR)"/extern_2.dll
```

Comment:
This syntax is incompatible with windows-gnu targets.

Reply:
I think I have a solution in place, but I'm having trouble getting a windows-gnu toolchain set up locally to test it -- creating an exe from a bin crate is failing for me even on (a slightly old version of) master and without attempting to use raw-dylib.  I may end up pushing what I have so I can work on other issues, but I'm not yet comfortable calling this issue resolved.

Reply:
We've decided not to support raw-dylib on windows-gnu as part of this PR, so I've marked this particular test as windows-msvc only.